akka-stream/src/main/scala/akka/stream/impl/PhasedFusingActorMaterializer.scala

```diff
@@ -659,7 +665,7 @@ private final case class SavedIslandData(
   def createPublisher(out: OutPort, logic: M): Publisher[Any]
 
   @InternalStableApi
-  def takePublisher(slot: Int, publisher: Publisher[Any]): Unit
+  def takePublisher(slot: Int, publisher: Publisher[Any], publisherAttributes: Attributes = Attributes.none): Unit
```

Comment:
We don't want default parameter value here, it complicates what is actually ending up in the bytecode (and therefore making keeping binary compatibility tricky), better be explicit about `none` where it is called.